neo/rawio/neuralynxrawio.py

```diff
@@ -297,7 +322,27 @@ def _get_signal_size(self, block_index, seg_index, channel_indexes):
     def _get_signal_t_start(self, block_index, seg_index, channel_indexes):
         return self._sigs_t_start[seg_index] - self.global_t_start
 
-    def _get_analogsignal_chunk(self, block_index, seg_index, i_start, i_stop, channel_indexes):
+    def _get_analogsignal_chunk(self, block_index, seg_index, i_start, i_stop, channel_indices):
```

Comment:
This channel_indexes>channel_indices could be globally a good idea.
But dor consistency we would need to change all IOs.
I propose to go back for this PR to channel_indexes because it is a very scondary change change.

@apdavison : should also make some comment sof this naming choice.

Reply:
I agree with @samuelgarcia, the naming discussion should be solved in another issue.

Reply:
OK, I just pushed a commit to this branch and pull request which reverts to channel_indexes, per convention.